### ISSUE TITLE
Improve crash handling and reporting

### ISSUE THREAD

**Issue by sawilde** [OpenCover/opencover#246]
This is not been an OpenCover strong point - we dump the error but don't really push the user to report it.
Was thinking of adding a reporting mechanism to make it easier e.g. look into using a nuget package like one of these perhaps http://www.nuget.org/packages?q=crash

**Comment by molnargab** [comment]
[soygul/NBug](https://github.com/soygul/NBug) is also worth for checking.

**Comment by sawilde** [comment]
@molnargab could be handy - is it possible to disable the dialogs - or to make the display work for console? OpenCover is often used without a user option so we may want to allow people to opt-in e.g. -autosubmitcrashreports...

**Comment by sawilde** [comment]
I just spiked a solution using http://crashreporterdotnet.codeplex.com/ - as it pushes the data to [DrDump](https://drdump.com/AppVersion.aspx?AppVersionID=1669) as a central location.

However I had to do some `hacking` as developers had not signed the assembly and no way to disable the UI so it was either use their UI or hack. However the [licence](http://crashreporterdotnet.codeplex.com/license) does not prohibit us in any way.
